integer/integer.go

```diff
@@ -16,7 +16,29 @@ limitations under the License.
 
 package integer
 
+// number is a constraint that permits any signed type, or any unsigned type
+type number interface {
+	~int | ~int8 | ~int16 | ~int32 | ~int64 | ~uint | ~uint8 | ~uint16 | ~uint32 | ~uint64 | ~uintptr
+}
```

Comment:
Introducing a user-defined constraint here will continue breaking CI till https://github.com/golang/go/issues/60911 is fixed.
See: https://github.com/kubernetes/utils/issues/284